pkg/filter/expr_filter.go

```diff
@@ -382,16 +378,16 @@ func (f *dmlExprFilter) addRule(
 }
 
 // verify checks if all rules in this filter is valid.
-func (f *dmlExprFilter) verify(tableInfos []*model.TableInfo) error {
-	for _, rule := range f.rules {
-		err := rule.verify(tableInfos)
-		if err != nil {
-			log.Error("failed to verify expression filter rule", zap.Error(err))
-			return errors.Trace(err)
-		}
-	}
-	return nil
-}
+// func (f *dmlExprFilter) verify(tableInfos []*model.TableInfo) error {
```

Comment:
Why did you comment this function?

Reply:
I’ll add the filter verification in another PR, since this part requires some additional changes.